test/test_autograd.py

```diff
@@ -2658,6 +2658,48 @@ def test_profiler_unboxed_only(self):
         with torch.autograd.profiler.profile() as prof:
             x.resize_([3, 2])
 
+    def test_profiler_propagation(self):
```

Comment:
Multithreaded autograd (https://github.com/pytorch/pytorch/pull/33157) landed some time ago, not sure if there's any additional testing we need to do to ensure that all backwards ops are profiled correctly with this change.

Reply:
um, we do have tests in place to test for autograd profiling

Reply:
how do you specifically enable multithreaded autograd? I can add it into existing test